.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - "2.1.5"
   - "2.2.0"
```

Comment:
Travis seems to explicitly want >= 2.2.5.  That's what we use for testing chef-client anyway...  so that sounds good to me.